Bottom: 68231d1b512e0f4fabd602fc53e98dbfa110504e
Top:    846f25c3c1b3601d504e618680abfea3b2ad164a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-23 23:37:58 +0000

Remove remaining RCS $Id$ identifiers left over from our CVS days

(Including a couple of $Id magic strings from files we got from
potrace, which are used by the gcode exporter).

NB: This alters the BOM and XY exporter output, as they previously
    embedded the (non-expanded since we moved to git) $Id$ string
    in their output.


---

diff --git a/Makefile.am b/Makefile.am
index c6927f1..6f9a94d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,4 @@
 ## -*- makefile -*-
-## $Id$
 ##
 ## Top level automake file for PCB
 
diff --git a/README.cvs_branches b/README.cvs_branches
index 41bf60a..00c025f 100644
--- a/README.cvs_branches
+++ b/README.cvs_branches
@@ -1,6 +1,3 @@
-# $Id$
-#
-
 usermenu - cvs branch where Dan is working on allowing the gtk HID
            to load its menus from a menu resource file like the lesstif
 	   HID.  No guarantees that this branch will even build, much less
diff --git a/README_FILES/Makefile.am b/README_FILES/Makefile.am
index ca74b2d..78f0801 100644
--- a/README_FILES/Makefile.am
+++ b/README_FILES/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 EXTRA_DIST= \
 	CHANGES \
 	Tools \
diff --git a/acinclude.m4 b/acinclude.m4
index 9ab3758..dab11df 100644
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -1,5 +1,3 @@
-dnl $Id$
-dnl
 dnl the FC_* macros were copied from the freeciv program.  The use here
 dnl is to figure out if we need -DNARROWPROTO and the correct setting
 dnl for FUNCPROTO.  Without these set right, it has been observed that
diff --git a/autogen.sh b/autogen.sh
index 166a3d0..0de878c 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -1,7 +1,5 @@
 #! /bin/sh
 #
-# $Id$
-#
 # Run the various GNU autotools to bootstrap the build
 # system.  Should only need to be done once.
 
diff --git a/doc/ascii2texi.awk b/doc/ascii2texi.awk
index fd0e415..b8e3a72 100755
--- a/doc/ascii2texi.awk
+++ b/doc/ascii2texi.awk
@@ -1,7 +1,4 @@
 #!/usr/bin/awk -f
-#
-# $Id$
-#
 
 BEGIN {
 	first = 1;
@@ -23,7 +20,6 @@ BEGIN {
 first == 1 {
 	first = 0;
 	printf("@c Generated file.  Do not edit directly\n");
-	printf("@c $" "Id" "$\n");
 	printf("@multitable @columnfractions ");
 	for(i = 1 ; i <= 2*ncol ; i = i + 1) {
 		printf("%.3g ", 0.5 / ncol);
diff --git a/doc/eps2png b/doc/eps2png
index d235dc4..893b7a3 100755
--- a/doc/eps2png
+++ b/doc/eps2png
@@ -1,7 +1,5 @@
 #!/usr/bin/perl
 
-my $RCS_Id = '$Id$ ';
-
 # Author          : Johan Vromans
 # Created On      : Tue Sep 15 15:59:04 1992
 # Last Modified By: Johan Vromans
diff --git a/doc/extract-docs b/doc/extract-docs
index a25de8e..1d0759e 100755
--- a/doc/extract-docs
+++ b/doc/extract-docs
@@ -1,8 +1,6 @@
 #!/usr/bin/perl
 # -*- perl -*-
 #
-# $Id$
-#
 #################################################################
 # This script extracts special comments from the source. It assembles
 # them in texinfo files that are included in the manual.  
diff --git a/doc/fractional_size.tab b/doc/fractional_size.tab
index eb68f6f..2687519 100644
--- a/doc/fractional_size.tab
+++ b/doc/fractional_size.tab
@@ -1,6 +1,3 @@
-# $Id$
-#
-
 1/64		.0156
 1/32		.0313
 3/64		.0469
diff --git a/doc/gs/gs.texi b/doc/gs/gs.texi
index b3cda3f..744642d 100644
--- a/doc/gs/gs.texi
+++ b/doc/gs/gs.texi
@@ -1,5 +1,4 @@
 \input texinfo    @c -*-texinfo-*-
-@comment RCS: $Id$
 @comment %**start of header
 @setfilename gs.info
 @settitle Getting Started With PCB
diff --git a/doc/gs/print-eps.scm b/doc/gs/print-eps.scm
index 427267d..4538610 100644
--- a/doc/gs/print-eps.scm
+++ b/doc/gs/print-eps.scm
@@ -1,5 +1,3 @@
-;; $Id$
-;;
 ;; This file may be used to print gschem schematics from the
 ;; command line.  Typical usage is:
 ;;
diff --git a/doc/ideas/database.txt b/doc/ideas/database.txt
index 380d1c9..d69139b 100644
--- a/doc/ideas/database.txt
+++ b/doc/ideas/database.txt
@@ -1,7 +1,3 @@
-# $Id$
-#
-#
-
 Please note that this document is a work in progress and
 currently represents a proposal for a future enhancement.
 The goal of this document is to define what changes need
diff --git a/doc/ideas/keepouts.txt b/doc/ideas/keepouts.txt
index 741d696..abfb483 100644
--- a/doc/ideas/keepouts.txt
+++ b/doc/ideas/keepouts.txt
@@ -1,7 +1,3 @@
-# $Id$
-#
-#
-
 Please note that this document is a work in progress and
 currently represents a proposal for a future enhancement.
 The goal of this document is to define what the keepout
diff --git a/doc/letter_size.tab b/doc/letter_size.tab
index 9d7459c..29b633b 100644
--- a/doc/letter_size.tab
+++ b/doc/letter_size.tab
@@ -1,6 +1,3 @@
-# $Id$
-#
-
 A		.2340
 B		.2380
 C		.2420
diff --git a/doc/metric_size.tab b/doc/metric_size.tab
index 3db5222..cfa464e 100644
--- a/doc/metric_size.tab
+++ b/doc/metric_size.tab
@@ -1,6 +1,3 @@
-# $Id$
-#
-
 0.20_mm  	.00787
 0.25_mm 	.00984
 0.30_mm  	.0118
diff --git a/doc/pcb.texi b/doc/pcb.texi
index 850b748..db214a2 100644
--- a/doc/pcb.texi
+++ b/doc/pcb.texi
@@ -1,5 +1,4 @@
 \input texinfo    @c -*-texinfo-*-
-@comment RCS: $Id$
 @comment %**start of header
 @setfilename pcb.info
 @settitle Pcb
diff --git a/doc/refcard.tex b/doc/refcard.tex
index 8072351..a735e1c 100644
--- a/doc/refcard.tex
+++ b/doc/refcard.tex
@@ -1,4 +1,3 @@
-% $Id$
 %
 %                            COPYRIGHT
 %
diff --git a/doc/wire_size.tab b/doc/wire_size.tab
index d72997f..8f57752 100644
--- a/doc/wire_size.tab
+++ b/doc/wire_size.tab
@@ -1,6 +1,3 @@
-# $Id$
-#
-
 97		.0059
 96		.0063
 95		.0067
diff --git a/doc/wishlist.txt b/doc/wishlist.txt
index e462abe..737a06d 100644
--- a/doc/wishlist.txt
+++ b/doc/wishlist.txt
@@ -1,6 +1,3 @@
-#  $Id$
-#
-
 This file is the start of a wish list of features/improvements in no particular
 order.
 
diff --git a/example/Makefile.am b/example/Makefile.am
index 9f7b034..6a14876 100644
--- a/example/Makefile.am
+++ b/example/Makefile.am
@@ -1,5 +1,3 @@
-## $Id$
-
 examplesdir=	@docdir@/examples
 examples_DATA=	${EXAMPLES}
 
diff --git a/example/libraries/Makefile.am b/example/libraries/Makefile.am
index 21392aa..9a33712 100644
--- a/example/libraries/Makefile.am
+++ b/example/libraries/Makefile.am
@@ -1,5 +1,3 @@
-## $Id$
-
 examplesdir=	@docdir@/examples/libraries
 examples_DATA=	${EXAMPLES}
 
diff --git a/example/libraries/example.inc b/example/libraries/example.inc
index 2ee0ec6..32c43b4 100644
--- a/example/libraries/example.inc
+++ b/example/libraries/example.inc
@@ -1,5 +1,4 @@
 # -*- m4 -*-
-# $Id$
 #
 # Example .inc file
 
diff --git a/example/libraries/example.m4 b/example/libraries/example.m4
index 2253a3c..ff2641d 100644
--- a/example/libraries/example.m4
+++ b/example/libraries/example.m4
@@ -1,6 +1,5 @@
 divert(-1)
 # -*- m4 -*-
-# $Id$
 #
 define(`Description_my_RC0402', 
   ``Standard SMT resistor/capacitor (0402)'')
diff --git a/globalconst.h b/globalconst.h
index afba0d4..e0779d5 100755
--- a/globalconst.h
+++ b/globalconst.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* global constants
diff --git a/lib/CreateLibrary.sh.in b/lib/CreateLibrary.sh.in
index 13ffa5e..6dd4f39 100755
--- a/lib/CreateLibrary.sh.in
+++ b/lib/CreateLibrary.sh.in
@@ -23,7 +23,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # creates a library
 #
diff --git a/lib/CreateLibraryContents.sh.in b/lib/CreateLibraryContents.sh.in
index ccba4a3..8f38224 100755
--- a/lib/CreateLibraryContents.sh.in
+++ b/lib/CreateLibraryContents.sh.in
@@ -23,7 +23,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # create all objects as defined in passed list_files
 #
diff --git a/lib/ListLibraryContents.sh.in b/lib/ListLibraryContents.sh.in
index cee49df..9e453b7 100755
--- a/lib/ListLibraryContents.sh.in
+++ b/lib/ListLibraryContents.sh.in
@@ -23,7 +23,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # list the contents of a library
 #
diff --git a/lib/QueryLibrary.sh.in b/lib/QueryLibrary.sh.in
index b5bed20..f053ced 100755
--- a/lib/QueryLibrary.sh.in
+++ b/lib/QueryLibrary.sh.in
@@ -23,7 +23,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # query a library
 #
diff --git a/lib/TTL_74xx_DIL.list b/lib/TTL_74xx_DIL.list
index e51b4bc..7fbaac4 100755
--- a/lib/TTL_74xx_DIL.list
+++ b/lib/TTL_74xx_DIL.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/TTL_74xx_DIL.m4 b/lib/TTL_74xx_DIL.m4
index df52b7f..ca3fe0f 100755
--- a/lib/TTL_74xx_DIL.m4
+++ b/lib/TTL_74xx_DIL.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 define(`Description_7400_dil', ``4 dual-NAND'')
 define(`Param1_7400_dil', 14)
diff --git a/lib/amp.inc b/lib/amp.inc
index cca93b9..e1fcf55 100644
--- a/lib/amp.inc
+++ b/lib/amp.inc
@@ -1,6 +1,5 @@
 # -*- m4 -*-
 #
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/amphenol.inc b/lib/amphenol.inc
index e41f874..b88ed39 100755
--- a/lib/amphenol.inc
+++ b/lib/amphenol.inc
@@ -1,6 +1,5 @@
 # -*- m4 -*-
 #
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/amphenol.list b/lib/amphenol.list
index 1393b15..d2690d2 100755
--- a/lib/amphenol.list
+++ b/lib/amphenol.list
@@ -1,4 +1,3 @@
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/amphenol.m4 b/lib/amphenol.m4
index fb499cf..c2c4f3c 100755
--- a/lib/amphenol.m4
+++ b/lib/amphenol.m4
@@ -1,5 +1,4 @@
 divert(-1)
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/bourns.inc b/lib/bourns.inc
index 0ebc298..23eee32 100644
--- a/lib/bourns.inc
+++ b/lib/bourns.inc
@@ -1,6 +1,5 @@
 # -*- m4 -*-
 #
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/candk.inc b/lib/candk.inc
index 94e59b9..6ad6072 100644
--- a/lib/candk.inc
+++ b/lib/candk.inc
@@ -1,6 +1,5 @@
 # -*- m4 -*-
 #
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/common.m4 b/lib/common.m4
index bc619c8..4c1f494 100644
--- a/lib/common.m4
+++ b/lib/common.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # common defines for packages
 #
diff --git a/lib/connector.inc b/lib/connector.inc
index 3bd8195..689f204 100755
--- a/lib/connector.inc
+++ b/lib/connector.inc
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # connector packages
 
diff --git a/lib/connector.list b/lib/connector.list
index d38e622..0af58a8 100755
--- a/lib/connector.list
+++ b/lib/connector.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/connector.m4 b/lib/connector.m4
index a264c19..ae1575c 100755
--- a/lib/connector.m4
+++ b/lib/connector.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 define(`Description_connector2', `connector 2x1 pins')
 define(`Param1_connector2', 1)
diff --git a/lib/crystal.list b/lib/crystal.list
index ad9c265..da0c7b4 100755
--- a/lib/crystal.list
+++ b/lib/crystal.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/crystal.m4 b/lib/crystal.m4
index bf2979d..cf7e4b2 100755
--- a/lib/crystal.m4
+++ b/lib/crystal.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 define(`Description_crystal_300', `crystal 300mil')
 define(`Param1_crystal_300', 300)
diff --git a/lib/cts.inc b/lib/cts.inc
index cf0b648..e1f281c 100644
--- a/lib/cts.inc
+++ b/lib/cts.inc
@@ -1,6 +1,5 @@
 # -*- m4 -*-
 #
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/dil.inc b/lib/dil.inc
index d1cde4a..6a75e0a 100755
--- a/lib/dil.inc
+++ b/lib/dil.inc
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # DIL packages
 
diff --git a/lib/geda.inc b/lib/geda.inc
index d25dd1d..985c18a 100644
--- a/lib/geda.inc
+++ b/lib/geda.inc
@@ -1,5 +1,4 @@
 # -*- m4 -*-
-#   $Id$
 #
 #   gEDA compatible footprint names
 #
diff --git a/lib/gen_geda_list.awk b/lib/gen_geda_list.awk
index 0770b47..fc73e5a 100755
--- a/lib/gen_geda_list.awk
+++ b/lib/gen_geda_list.awk
@@ -1,7 +1,5 @@
 #!/usr/bin/awk -f
 #
-# $Id$
-#
 # Script to regenerate geda.list from geda.inc
 #
 # Usage:
diff --git a/lib/gen_geda_m4.awk b/lib/gen_geda_m4.awk
index 88994de..ccbf4d8 100755
--- a/lib/gen_geda_m4.awk
+++ b/lib/gen_geda_m4.awk
@@ -1,7 +1,5 @@
 #!/usr/bin/awk -f
 #
-# $Id$
-#
 # Script to regenerate geda.m4 from geda.inc
 #
 # Usage:
diff --git a/lib/gen_list.awk b/lib/gen_list.awk
index 5a9e8cd..b3d672e 100755
--- a/lib/gen_list.awk
+++ b/lib/gen_list.awk
@@ -1,7 +1,5 @@
 #!/usr/bin/awk -f
 #
-# $Id$
-#
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/gen_m4.awk b/lib/gen_m4.awk
index 2b16867..544f686 100755
--- a/lib/gen_m4.awk
+++ b/lib/gen_m4.awk
@@ -1,7 +1,5 @@
 #!/usr/bin/awk -f
 #
-# $Id$
-#
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/generic.list b/lib/generic.list
index bc31768..2fc70ab 100755
--- a/lib/generic.list
+++ b/lib/generic.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/generic.m4 b/lib/generic.m4
index cd19fcd..a08cac0 100755
--- a/lib/generic.m4
+++ b/lib/generic.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 define(`Description_generic8_dil', ``generic'')
 define(`Param1_generic8_dil', 8)
diff --git a/lib/jerry.list b/lib/jerry.list
index 903e853..6d03af9 100644
--- a/lib/jerry.list
+++ b/lib/jerry.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 
 #  Arcade machine parts needed by Scott "Jerry" Lawrence
 #   jsl@absynth.com
diff --git a/lib/jerry.m4 b/lib/jerry.m4
index f3f742c..28dd30f 100644
--- a/lib/jerry.m4
+++ b/lib/jerry.m4
@@ -23,8 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
-#
 
 #  Arcade machine parts needed by Scott "Jerry" Lawrence
 #   jsl@absynth.com
diff --git a/lib/johnstech.inc b/lib/johnstech.inc
index 926a3ad..0a11fee 100644
--- a/lib/johnstech.inc
+++ b/lib/johnstech.inc
@@ -1,5 +1,4 @@
 # -*- m4 -*-
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/linear.list b/lib/linear.list
index 80eabe3..ab120fd 100755
--- a/lib/linear.list
+++ b/lib/linear.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/linear.m4 b/lib/linear.m4
index ba01501..5b8e695 100755
--- a/lib/linear.m4
+++ b/lib/linear.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 define(`Description_MAX222_dil', `high-speed dual RS232 driver w. shutdown')
 define(`Param1_MAX222_dil', 18)
diff --git a/lib/logic.list b/lib/logic.list
index 8777e57..9f59eb5 100755
--- a/lib/logic.list
+++ b/lib/logic.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/logic.m4 b/lib/logic.m4
index 517767a..15a2cb1 100755
--- a/lib/logic.m4
+++ b/lib/logic.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # ------------------------------------------------------------------------
 # based on mail by Olaf Kaluza (olaf@criseis.ruhr.de)
diff --git a/lib/lsi.list b/lib/lsi.list
index d805435..b98afdf 100755
--- a/lib/lsi.list
+++ b/lib/lsi.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/lsi.m4 b/lib/lsi.m4
index c495f90..99dd15c 100755
--- a/lib/lsi.m4
+++ b/lib/lsi.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # ----------------------------------------------------------------------
 # several different microcontrollers from the PIC family
diff --git a/lib/m4lib_to_newlib.sh b/lib/m4lib_to_newlib.sh
index 09499b6..6ca4b81 100644
--- a/lib/m4lib_to_newlib.sh
+++ b/lib/m4lib_to_newlib.sh
@@ -1,7 +1,4 @@
 #!/bin/sh
-#
-# $Id$
-#
 
 # This script is used to extract all elements from an "oldlib" (M4)
 # style library and place them in individual "newlib" style files.
diff --git a/lib/memory.list b/lib/memory.list
index 00e7b88..ed58e7f 100755
--- a/lib/memory.list
+++ b/lib/memory.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/memory.m4 b/lib/memory.m4
index 8bdceeb..7af73fb 100755
--- a/lib/memory.m4
+++ b/lib/memory.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 define(`Description_27512_dil', `EPROM 64Kx8')
 define(`Param1_27512_dil', 28)
diff --git a/lib/minicircuits.inc b/lib/minicircuits.inc
index 22174a5..9ec6218 100644
--- a/lib/minicircuits.inc
+++ b/lib/minicircuits.inc
@@ -1,6 +1,5 @@
 # -*- m4 -*-
 #
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/misc.inc b/lib/misc.inc
index 380d511..f34a3fc 100755
--- a/lib/misc.inc
+++ b/lib/misc.inc
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # misc packages
 #
diff --git a/lib/nichicon.inc b/lib/nichicon.inc
index 545e1d4..f96c151 100644
--- a/lib/nichicon.inc
+++ b/lib/nichicon.inc
@@ -1,6 +1,5 @@
 # -*- m4 -*-
 #
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/optek.inc b/lib/optek.inc
index 943f0c4..711ecde 100644
--- a/lib/optek.inc
+++ b/lib/optek.inc
@@ -1,6 +1,5 @@
 # -*- m4 -*-
 #
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/optical.list b/lib/optical.list
index 8fa292b..1a78dfb 100755
--- a/lib/optical.list
+++ b/lib/optical.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/optical.m4 b/lib/optical.m4
index 92b6a17..24a7f52 100755
--- a/lib/optical.m4
+++ b/lib/optical.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # most (all at the beginning) of the data was provided by
 # Volker Bosch (bosch@iema.e-technik.uni-stuttgart.de)
diff --git a/lib/panasonic.inc b/lib/panasonic.inc
index 5c3b85a..b539cd6 100644
--- a/lib/panasonic.inc
+++ b/lib/panasonic.inc
@@ -1,6 +1,5 @@
 # -*- m4 -*-
 #
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/pci.inc b/lib/pci.inc
index 762749e..23d9189 100644
--- a/lib/pci.inc
+++ b/lib/pci.inc
@@ -1,5 +1,3 @@
-# $Id$
-#
 # Definitions for PCI boards
 # by D.J. Barrow dj_barrow@ariasoft.ie
 #
diff --git a/lib/pci.list b/lib/pci.list
index 0ddcb59..44f9fbd 100644
--- a/lib/pci.list
+++ b/lib/pci.list
@@ -1,6 +1,3 @@
-# $Id$
-#
-
 PCI5V_AVE_HEIGHT:PCI5V_AVE_HEIGHT:PCI5V_AVE_HEIGHT
 PCI5V_MIN_HEIGHT:PCI5V_MIN_HEIGHT:PCI5V_MIN_HEIGHT
 PCI5V_MAX_HEIGHT:PCI5V_MAX_HEIGHT:PCI5V_MAX_HEIGHT
diff --git a/lib/pci.m4 b/lib/pci.m4
index 632ec7d..4dfa176 100644
--- a/lib/pci.m4
+++ b/lib/pci.m4
@@ -1,5 +1,4 @@
 divert(-1)
-# $Id$
 #
 # Definitions for PCI boards
 # by D.J. Barrow dj_barrow@ariasoft.ie
diff --git a/lib/plcc.inc b/lib/plcc.inc
index 5e1a1dc..cbe7458 100755
--- a/lib/plcc.inc
+++ b/lib/plcc.inc
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # PLCC packages
 #
diff --git a/lib/png_diff.sh b/lib/png_diff.sh
index 478c19c..ee557fd 100755
--- a/lib/png_diff.sh
+++ b/lib/png_diff.sh
@@ -1,7 +1,5 @@
 #!/bin/sh
 #
-# $Id$
-#
 # Copyright (c) 2003, 2004, 2005, 2006, 2007 Dan McMahill
 
 #  This program is free software; you can redistribute it and/or modify
diff --git a/lib/qfn.inc b/lib/qfn.inc
index d8fb50f..5710cc6 100644
--- a/lib/qfn.inc
+++ b/lib/qfn.inc
@@ -1,5 +1,4 @@
 # -*- m4 -*-
-# $Id$
 #                            COPYRIGHT
 #
 #   PCB, interactive printed circuit board design
diff --git a/lib/qfp-ui.in b/lib/qfp-ui.in
index f6261a1..fe0b8d0 100755
--- a/lib/qfp-ui.in
+++ b/lib/qfp-ui.in
@@ -1,6 +1,5 @@
 #!@WISH@ -f
 
-# $Id$
 #
 # User Interface that generates custom QFP and SOIC packages for pcb-1.6.3
 # Invoked from a line like
diff --git a/lib/qfp.inc b/lib/qfp.inc
index 0ceaeed..a0598d4 100644
--- a/lib/qfp.inc
+++ b/lib/qfp.inc
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 #  QFP packages
 #
diff --git a/lib/qfp2.inc b/lib/qfp2.inc
index 04f6aa2..130f237 100644
--- a/lib/qfp2.inc
+++ b/lib/qfp2.inc
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 #  QFP packages
 #
diff --git a/lib/qfpdj.inc b/lib/qfpdj.inc
index 51e30b1..98c5d5b 100644
--- a/lib/qfpdj.inc
+++ b/lib/qfpdj.inc
@@ -24,7 +24,6 @@
 #   Improvments by D.J. Barrow dj_barrow@ariasoft.ie
 #   Further improved by W. Kazubski <wk@ire.pw.edu.pl>
 # 
-#   RCS: $Id$
 #
 #  QFP packages
 #
diff --git a/lib/resistor_0.25W.list b/lib/resistor_0.25W.list
index 3c68234..71e27dc 100755
--- a/lib/resistor_0.25W.list
+++ b/lib/resistor_0.25W.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/resistor_0.25W.m4 b/lib/resistor_0.25W.m4
index 3487ab6..b83a624 100755
--- a/lib/resistor_0.25W.m4
+++ b/lib/resistor_0.25W.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 define(`Description_r_025', ``R 0.25W'')
 
diff --git a/lib/resistor_adjust.list b/lib/resistor_adjust.list
index a211622..b6d2657 100644
--- a/lib/resistor_adjust.list
+++ b/lib/resistor_adjust.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/resistor_adjust.m4 b/lib/resistor_adjust.m4
index ec13611..10def9f 100644
--- a/lib/resistor_adjust.m4
+++ b/lib/resistor_adjust.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # most (all at the beginning) of the data was provided by
 # Volker Bosch (bosch@iema.e-technik.uni-stuttgart.de)
diff --git a/lib/resistor_array.list b/lib/resistor_array.list
index f659ba8..32ed7af 100755
--- a/lib/resistor_array.list
+++ b/lib/resistor_array.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/resistor_array.m4 b/lib/resistor_array.m4
index bdf96e0..a939410 100755
--- a/lib/resistor_array.m4
+++ b/lib/resistor_array.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 
 # ----------------------------------------------------------------------
 # resistor array without common pin
diff --git a/lib/rules.inc b/lib/rules.inc
index aee42f9..244e592 100644
--- a/lib/rules.inc
+++ b/lib/rules.inc
@@ -1,5 +1,4 @@
 #
-#   $Id$
 #
 #   design rule definitions --
 #   tune design rules here
diff --git a/lib/smt.inc b/lib/smt.inc
index ace76a0..eba644a 100644
--- a/lib/smt.inc
+++ b/lib/smt.inc
@@ -1,5 +1,4 @@
 # -*- m4 -*-
-#   $Id$
 #
 #   surface mounted components
 #
diff --git a/lib/texas_inst_amplifier.list b/lib/texas_inst_amplifier.list
index 531aaad..4776a48 100755
--- a/lib/texas_inst_amplifier.list
+++ b/lib/texas_inst_amplifier.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/texas_inst_amplifier.m4 b/lib/texas_inst_amplifier.m4
index 01c2ea5..47572f1 100755
--- a/lib/texas_inst_amplifier.m4
+++ b/lib/texas_inst_amplifier.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 define(`Description_TL070_dil', `Single low noise JFET input operational amplifier')
 define(`Param1_TL070_dil', 8)
diff --git a/lib/texas_inst_voltage_reg.list b/lib/texas_inst_voltage_reg.list
index 862cea9..100e641 100644
--- a/lib/texas_inst_voltage_reg.list
+++ b/lib/texas_inst_voltage_reg.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 #
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/texas_inst_voltage_reg.m4 b/lib/texas_inst_voltage_reg.m4
index a6fdb2f..5a1052c 100644
--- a/lib/texas_inst_voltage_reg.m4
+++ b/lib/texas_inst_voltage_reg.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 #
-#   RCS: $Id$
 #
 #
 define(`Description_uA7905C',`-5V 1.5A voltage regulator')
diff --git a/lib/transistor.list b/lib/transistor.list
index 2e7d339..f814079 100755
--- a/lib/transistor.list
+++ b/lib/transistor.list
@@ -22,7 +22,6 @@
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # mask package values...
 #
diff --git a/lib/transistor.m4 b/lib/transistor.m4
index fa33af6..fa112fe 100755
--- a/lib/transistor.m4
+++ b/lib/transistor.m4
@@ -23,7 +23,6 @@ divert(-1)
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
 # 
-#   RCS: $Id$
 #
 # most data by Volker Bosch (bosch@iema.e-technik.uni-stuttgart.de
 #
diff --git a/newlib/2_pin_thru-hole_packages/Makefile.am b/newlib/2_pin_thru-hole_packages/Makefile.am
index 18f59ed..bc67252 100644
--- a/newlib/2_pin_thru-hole_packages/Makefile.am
+++ b/newlib/2_pin_thru-hole_packages/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/2_pin_thru-hole_packages
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS= \
diff --git a/newlib/Makefile.am b/newlib/Makefile.am
index e38b946..da5dc95 100644
--- a/newlib/Makefile.am
+++ b/newlib/Makefile.am
@@ -1,5 +1,3 @@
-## $Id$
-##
 ## Top level 'newlib' automake file
 
 SUBDIRS=\
diff --git a/newlib/connectors/Makefile.am b/newlib/connectors/Makefile.am
index b1d56c8..b261d4d 100644
--- a/newlib/connectors/Makefile.am
+++ b/newlib/connectors/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/connectors
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS= \
diff --git a/newlib/crystal/Makefile.am b/newlib/crystal/Makefile.am
index a45bf12..f91a2d2 100644
--- a/newlib/crystal/Makefile.am
+++ b/newlib/crystal/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/crystal
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS= \
diff --git a/newlib/electro-optics/Makefile.am b/newlib/electro-optics/Makefile.am
index 5003ae8..c84c7fd 100644
--- a/newlib/electro-optics/Makefile.am
+++ b/newlib/electro-optics/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/electro-optics
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS= \
diff --git a/newlib/headers/Makefile.am b/newlib/headers/Makefile.am
index 6a196bf..b543e09 100644
--- a/newlib/headers/Makefile.am
+++ b/newlib/headers/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/headers
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS= \
diff --git a/newlib/keystone/Makefile.am b/newlib/keystone/Makefile.am
index 295f064..e0cde83 100644
--- a/newlib/keystone/Makefile.am
+++ b/newlib/keystone/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/keystone
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS= \
diff --git a/newlib/msp430/Makefile.am b/newlib/msp430/Makefile.am
index ae9ac11..08471b2 100644
--- a/newlib/msp430/Makefile.am
+++ b/newlib/msp430/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/msp430
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS= \
diff --git a/newlib/not_vetted_ingo/Makefile.am b/newlib/not_vetted_ingo/Makefile.am
index 580e9c8..1b8f3fe 100644
--- a/newlib/not_vetted_ingo/Makefile.am
+++ b/newlib/not_vetted_ingo/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/not_vetted_ingo
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS= \
diff --git a/newlib/sockets/Makefile.am b/newlib/sockets/Makefile.am
index 5cf859f..15b4e57 100644
--- a/newlib/sockets/Makefile.am
+++ b/newlib/sockets/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/sockets
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS=
diff --git a/newlib/tests/Makefile.am b/newlib/tests/Makefile.am
index 0be6654..356d9ba 100644
--- a/newlib/tests/Makefile.am
+++ b/newlib/tests/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 pcblibsubdir= @PCBTREEDIR@/tests
 pcblibsub_DATA= ${COMPONENTS}
 COMPONENTS= \
diff --git a/src/Makefile.am b/src/Makefile.am
index 7adb424..47035f6 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,6 +1,4 @@
 ## -*- makefile -*-
-## $Id$
-##
 
 SUBDIRS=	icons
 
diff --git a/src/action.h b/src/action.h
index 83089f3..4b93ef8 100644
--- a/src/action.h
+++ b/src/action.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for action routines
diff --git a/src/autoplace.c b/src/autoplace.c
index 0e74b66..e7cdee8 100644
--- a/src/autoplace.c
+++ b/src/autoplace.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -67,8 +65,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #define EXPANDRECTXY(r1, x1, y1, x2, y2) { \
   r1->X1=MIN(r1->X1, x1); r1->Y1=MIN(r1->Y1, y1); \
   r1->X2=MAX(r1->X2, x2); r1->Y2=MAX(r1->Y2, y2); \
diff --git a/src/autoplace.h b/src/autoplace.h
index 5133cf0..75e4c8f 100644
--- a/src/autoplace.h
+++ b/src/autoplace.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/autoroute.c b/src/autoroute.c
index 34e01dc..374b6ac 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -86,8 +84,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* #defines to enable some debugging output */
 /*
 #define ROUTE_VERBOSE
diff --git a/src/autoroute.h b/src/autoroute.h
index d659349..256a495 100644
--- a/src/autoroute.h
+++ b/src/autoroute.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/box.h b/src/box.h
index 1c8311a..943b316 100644
--- a/src/box.h
+++ b/src/box.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/buffer.c b/src/buffer.c
index ac2aa54..1f9293a 100644
--- a/src/buffer.c
+++ b/src/buffer.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -62,8 +60,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/buffer.h b/src/buffer.h
index e589fd4..6871216 100644
--- a/src/buffer.h
+++ b/src/buffer.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for buffer handling routines
diff --git a/src/change.c b/src/change.c
index 61cb335..aaad7a2 100644
--- a/src/change.c
+++ b/src/change.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -63,8 +61,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/change.h b/src/change.h
index 64c44ff..3fd896d 100644
--- a/src/change.h
+++ b/src/change.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes to change object properties
diff --git a/src/clip.c b/src/clip.c
index f38c660..e4fef34 100644
--- a/src/clip.c
+++ b/src/clip.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -45,8 +43,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* Clip the line to the clipBox
  * return true if something to be drawn
  * false if the whole thing is clipped
diff --git a/src/clip.h b/src/clip.h
index c06d64f..aaec5a8 100644
--- a/src/clip.h
+++ b/src/clip.h
@@ -23,7 +23,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for inserting points into objects
diff --git a/src/command.c b/src/command.c
index 963ed85..8055ec8 100644
--- a/src/command.c
+++ b/src/command.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -57,8 +55,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------- */
 
 /*  %start-doc actions 00macros
diff --git a/src/command.h b/src/command.h
index 5517993..5018f28 100644
--- a/src/command.h
+++ b/src/command.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for command execution
diff --git a/src/compat.c b/src/compat.c
index e3acf2f..6092764 100644
--- a/src/compat.c
+++ b/src/compat.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -34,8 +32,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #ifndef HAVE_EXPF
 float
 expf (float x)
diff --git a/src/compat.h b/src/compat.h
index 9afa02c..ff83161 100644
--- a/src/compat.h
+++ b/src/compat.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/const.h b/src/const.h
index 078f83f..7d732e8 100644
--- a/src/const.h
+++ b/src/const.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* global source constants
diff --git a/src/copy.c b/src/copy.c
index 29ac62f..193bb52 100644
--- a/src/copy.c
+++ b/src/copy.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -57,8 +55,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/copy.h b/src/copy.h
index e65c272..af0b7ff 100644
--- a/src/copy.h
+++ b/src/copy.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for copy routines
diff --git a/src/create.c b/src/create.c
index 8d8344c..8a1effd 100644
--- a/src/create.c
+++ b/src/create.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -59,8 +57,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------------
  * some local identifiers
  */
diff --git a/src/create.h b/src/create.h
index ccab272..faebe52 100644
--- a/src/create.h
+++ b/src/create.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for create routines
diff --git a/src/crosshair.c b/src/crosshair.c
index 177c9cc..055f390 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -1,4 +1,3 @@
-/* $Id$ */
 /*
  *                            COPYRIGHT
  *
@@ -52,8 +51,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 typedef struct
 {
   int x, y;
diff --git a/src/crosshair.h b/src/crosshair.h
index 8c42067..2dd5e7e 100644
--- a/src/crosshair.h
+++ b/src/crosshair.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for crosshair routines
diff --git a/src/data.c b/src/data.c
index ebd9b29..702d9ca 100644
--- a/src/data.c
+++ b/src/data.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -38,8 +36,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------------
  * some shared identifiers
  */
diff --git a/src/data.h b/src/data.h
index 502acdd..efb2ed1 100644
--- a/src/data.h
+++ b/src/data.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* common identifiers
diff --git a/src/djopt.c b/src/djopt.c
index 7b5a223..daeae0a 100644
--- a/src/djopt.c
+++ b/src/djopt.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -50,8 +48,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #ifndef HAVE_RINT
 #define rint(x)  (ceil((x) - 0.5))
 #endif
diff --git a/src/djopt.h b/src/djopt.h
index bc337af..15014c1 100644
--- a/src/djopt.h
+++ b/src/djopt.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/dolists.h b/src/dolists.h
index d16f055..3b7c663 100644
--- a/src/dolists.h
+++ b/src/dolists.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #undef REGISTER_ACTIONS
 #undef REGISTER_ATTRIBUTES
 #undef REGISTER_FLAGS
diff --git a/src/draw.c b/src/draw.c
index a06dde1..a11c6e2 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -61,8 +59,6 @@
 #define MAXINT (((unsigned int)(~0))>>1)
 #endif
 
-RCSID ("$Id$");
-
 #define	SMALL_SMALL_TEXT_SIZE	0
 #define	SMALL_TEXT_SIZE			1
 #define	NORMAL_TEXT_SIZE		2
diff --git a/src/draw.h b/src/draw.h
index d92dee5..cc0822f 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for drawing routines
diff --git a/src/drill.c b/src/drill.c
index a7a908b..efa9baa 100644
--- a/src/drill.c
+++ b/src/drill.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -41,11 +39,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
-
-
 /*
  * some local prototypes
  */
diff --git a/src/drill.h b/src/drill.h
index 862ff08..f155a72 100644
--- a/src/drill.h
+++ b/src/drill.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/error.c b/src/error.c
index f01420e..4e802a4 100644
--- a/src/error.c
+++ b/src/error.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -61,9 +59,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 #define utf8_dup_string(a,b) *(a) = strdup(b)
 
 /* ----------------------------------------------------------------------
diff --git a/src/error.h b/src/error.h
index 8409767..8cd99a2 100644
--- a/src/error.h
+++ b/src/error.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for error and debug functions
diff --git a/src/file.c b/src/file.c
index 92e719b..c157659 100644
--- a/src/file.c
+++ b/src/file.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -100,8 +98,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #if !defined(HAS_ATEXIT) && !defined(HAS_ON_EXIT)
 /* ---------------------------------------------------------------------------
  * some local identifiers for OS without an atexit() or on_exit()
diff --git a/src/find.c b/src/find.c
index cd2aff5..fa03ba7 100644
--- a/src/find.c
+++ b/src/find.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *
  *                            COPYRIGHT
@@ -102,11 +100,6 @@
 
 #undef DEBUG
 
-RCSID ("$Id$");
-
-
-
-
 /* ---------------------------------------------------------------------------
  * some local macros
  */
diff --git a/src/find.h b/src/find.h
index d2be7ac..1d54111 100644
--- a/src/find.h
+++ b/src/find.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes connection search routines
diff --git a/src/flags.c b/src/flags.c
index de77b68..d809ad5 100644
--- a/src/flags.c
+++ b/src/flags.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -42,8 +40,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static int
 FlagCurrentStyle (int dummy)
 {
diff --git a/src/fontmode.c b/src/fontmode.c
index dd64f63..f218e7c 100644
--- a/src/fontmode.c
+++ b/src/fontmode.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -52,8 +50,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* FIXME - we currently hardcode the grid and PCB size.  What we
    should do in the future is scan the font for its extents, and size
    the grid appropriately.  Also, when we convert back to a font, we
diff --git a/src/gather-actions b/src/gather-actions
index f024158..d8b9e9d 100644
--- a/src/gather-actions
+++ b/src/gather-actions
@@ -1,6 +1,5 @@
 #!/usr/bin/perl
 # -*- perl -*-
-#  $Id$
 #
 #                            COPYRIGHT
 #  PCB, interactive printed circuit board design
diff --git a/src/global.h b/src/global.h
index 029d63c..79c1456 100644
--- a/src/global.h
+++ b/src/global.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* definition of types
@@ -823,8 +822,6 @@ struct drc_violation_st
 #define ATTRIBUTE_UNUSED
 #endif
 
-#define RCSID(x) static char *rcsid  ATTRIBUTE_UNUSED = x
-
 /* ---------------------------------------------------------------------------
  * Macros called by various action routines to show usage or to report
  * a syntax error and fail
diff --git a/src/heap.c b/src/heap.c
index 57a8208..50b11c0 100644
--- a/src/heap.c
+++ b/src/heap.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -48,9 +46,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 /* define this for more thorough self-checking of data structures */
 #undef SLOW_ASSERTIONS
 
diff --git a/src/heap.h b/src/heap.h
index 381c7b2..85e6a53 100644
--- a/src/heap.h
+++ b/src/heap.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/hid/batch/batch.c b/src/hid/batch/batch.c
index ca8caf0..8b969fa 100644
--- a/src/hid/batch/batch.c
+++ b/src/hid/batch/batch.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -27,8 +25,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* This is a text-line "batch" HID, which exists for scripting and
    non-GUI needs.  */
 
diff --git a/src/hid/bom/bom.c b/src/hid/bom/bom.c
index b3529cd..e67450a 100644
--- a/src/hid/bom/bom.c
+++ b/src/hid/bom/bom.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -24,8 +22,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static HID_Attribute bom_options[] = {
 /* %start-doc options "8 BOM Creation"
 @ftable @code
diff --git a/src/hid/common/actions.c b/src/hid/common/actions.c
index 13bb129..abd86ae 100644
--- a/src/hid/common/actions.c
+++ b/src/hid/common/actions.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -20,8 +18,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static HID_Action **all_actions = 0;
 static int all_actions_sorted = 0;
 static int n_actions = 0;
diff --git a/src/hid/common/extents.c b/src/hid/common/extents.c
index 6cdc135..d970c77 100644
--- a/src/hid/common/extents.c
+++ b/src/hid/common/extents.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -20,8 +18,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #ifndef MAXINT
 #define MAXINT (((unsigned int)(~0))>>1)
 #endif
diff --git a/src/hid/common/flags.c b/src/hid/common/flags.c
index 92032f9..9c87a54 100644
--- a/src/hid/common/flags.c
+++ b/src/hid/common/flags.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -19,8 +17,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 typedef struct HID_FlagNode
 {
   struct HID_FlagNode *next;
diff --git a/src/hid/common/hidinit.c b/src/hid/common/hidinit.c
index 3695f7d..9f05800 100644
--- a/src/hid/common/hidinit.c
+++ b/src/hid/common/hidinit.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -38,8 +36,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #define HID_DEF(x) extern void hid_ ## x ## _init(void);
 #include "hid/common/hidlist.h"
 #undef HID_DEF
diff --git a/src/hid/common/hidnogui.c b/src/hid/common/hidnogui.c
index b08d90d..d772e18 100644
--- a/src/hid/common/hidnogui.c
+++ b/src/hid/common/hidnogui.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -16,8 +14,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* This is the "gui" that is installed at startup, and is used when
    there is no other real GUI to use.  For the most part, it just
    stops the application from (1) crashing randomly, and (2) doing
diff --git a/src/hid/gcode/curve.c b/src/hid/gcode/curve.c
index abf85d7..24b927c 100644
--- a/src/hid/gcode/curve.c
+++ b/src/hid/gcode/curve.c
@@ -2,7 +2,7 @@
    This file is part of Potrace. It is free software and it is covered
    by the GNU General Public License. See the file COPYING for details. */
 
-/* $Id: curve.c 147 2007-04-09 00:44:09Z selinger $ */
+/* curve.c 147 2007-04-09 00:44:09Z selinger */
 /* private part of the path and curve data structures */
 
 #include <stdio.h>
diff --git a/src/hid/gcode/decompose.c b/src/hid/gcode/decompose.c
index 0b42882..eeac028 100644
--- a/src/hid/gcode/decompose.c
+++ b/src/hid/gcode/decompose.c
@@ -2,7 +2,7 @@
    This file is part of Potrace. It is free software and it is covered
    by the GNU General Public License. See the file COPYING for details. */
 
-/* $Id: decompose.c 146 2007-04-09 00:43:46Z selinger $ */
+/* decompose.c 146 2007-04-09 00:43:46Z selinger */
 
 #include <stdio.h>
 #include <stdlib.h>
diff --git a/src/hid/gcode/decompose.h b/src/hid/gcode/decompose.h
index 333003e..9c11429 100644
--- a/src/hid/gcode/decompose.h
+++ b/src/hid/gcode/decompose.h
@@ -2,7 +2,7 @@
    This file is part of Potrace. It is free software and it is covered
    by the GNU General Public License. See the file COPYING for details. */
 
-/* $Id: decompose.h 147 2007-04-09 00:44:09Z selinger $ */
+/* decompose.h 147 2007-04-09 00:44:09Z selinger */
 
 #ifndef DECOMPOSE_H
 #define DECOMPOSE_H
diff --git a/src/hid/gcode/gcode.h b/src/hid/gcode/gcode.h
index ae9d7a0..e0db890 100644
--- a/src/hid/gcode/gcode.h
+++ b/src/hid/gcode/gcode.h
@@ -1,3 +1 @@
-/* $Id: nelma.h,v 1.2 2007/04/20 11:31:15 danmc Exp $ */
-
 extern HID gcode_hid;
diff --git a/src/hid/gcode/lists.h b/src/hid/gcode/lists.h
index 3513184..60d42e6 100644
--- a/src/hid/gcode/lists.h
+++ b/src/hid/gcode/lists.h
@@ -2,7 +2,7 @@
    This file is part of Potrace. It is free software and it is covered
    by the GNU General Public License. See the file COPYING for details. */
 
-/* $Id: lists.h 147 2007-04-09 00:44:09Z selinger $ */
+/* lists.h 147 2007-04-09 00:44:09Z selinger */
 
 #ifndef PCB_HID_GCODE_LISTS_H
 #define PCB_HID_GCODE_LISTS_H
diff --git a/src/hid/gcode/trace.c b/src/hid/gcode/trace.c
index 5d7c62e..9a3f743 100644
--- a/src/hid/gcode/trace.c
+++ b/src/hid/gcode/trace.c
@@ -4,7 +4,7 @@
    This file is part of Potrace. It is free software and it is covered
    by the GNU General Public License. See the file COPYING for details. */
 
-/* $Id: trace.c 147 2007-04-09 00:44:09Z selinger $ */
+/* trace.c 147 2007-04-09 00:44:09Z selinger */
 /* transform jaggy paths into smooth curves */
 
 #include <stdio.h>
diff --git a/src/hid/gcode/trace.h b/src/hid/gcode/trace.h
index d55fd3e..ead6d50 100644
--- a/src/hid/gcode/trace.h
+++ b/src/hid/gcode/trace.h
@@ -2,7 +2,7 @@
    This file is part of Potrace. It is free software and it is covered
    by the GNU General Public License. See the file COPYING for details. */
 
-/* $Id: trace.h 147 2007-04-09 00:44:09Z selinger $ */
+/* trace.h 147 2007-04-09 00:44:09Z selinger */
 
 #ifndef TRACE_H
 #define TRACE_H
diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 3973814..dd9c1c9 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -38,8 +36,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented Gerber function %s.\n", __FUNCTION__); abort()
 
 /*----------------------------------------------------------------------------*/
diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 9f4250d..8e6cf22 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -16,10 +14,6 @@
 #include <dmalloc.h>
 #endif
 
-
-RCSID ("$Id$");
-
-
 extern HID ghid_hid;
 
 /* Sets priv->u_gc to the "right" GC to use (wrt mask or window)
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index d372f8a..7471afc 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -33,9 +31,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 extern HID ghid_hid;
 
 static hidGC current_gc = NULL;
diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index cc474cd..d8bf02c 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -26,10 +24,6 @@
 #include <dmalloc.h>
 #endif
 
-
-RCSID ("$Id$");
-
-
 static void
 pan_common (GHidPort *port)
 {
diff --git a/src/hid/gtk/gtkhid.h b/src/hid/gtk/gtkhid.h
index f55845b..d6e9887 100644
--- a/src/hid/gtk/gtkhid.h
+++ b/src/hid/gtk/gtkhid.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifndef PCB_HID_GTK_GTKHID_H
 #define PCB_HID_GTK_GTKHID_H
 
diff --git a/src/hid/gtk/gui-command-window.c b/src/hid/gtk/gui-command-window.c
index d913cae..25b0154 100644
--- a/src/hid/gtk/gui-command-window.c
+++ b/src/hid/gtk/gui-command-window.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -42,8 +40,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static GtkWidget *command_window;
 static GtkWidget *combo_vbox;
 static GList *history_list;
diff --git a/src/hid/gtk/gui-config.c b/src/hid/gtk/gui-config.c
index 2a4fcd8..03fa15b 100644
--- a/src/hid/gtk/gui-config.c
+++ b/src/hid/gtk/gui-config.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -58,9 +56,6 @@
 
 extern int	MoveLayerAction(int argc, char **argv, int x, int y);
 
-
-RCSID ("$Id$");
-
 enum ConfigType
 {
   CONFIG_Boolean,
diff --git a/src/hid/gtk/gui-dialog-print.c b/src/hid/gtk/gui-dialog-print.c
index 5d7d003..b418688 100644
--- a/src/hid/gtk/gui-dialog-print.c
+++ b/src/hid/gtk/gui-dialog-print.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -43,8 +41,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static GtkWidget *export_dialog = NULL;
 
 static void
diff --git a/src/hid/gtk/gui-dialog.c b/src/hid/gtk/gui-dialog.c
index 1770bba..87ee906 100644
--- a/src/hid/gtk/gui-dialog.c
+++ b/src/hid/gtk/gui-dialog.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -42,8 +40,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------- */
 gchar *
 ghid_dialog_input (const char * prompt, const char * initial)
diff --git a/src/hid/gtk/gui-drc-window.c b/src/hid/gtk/gui-drc-window.c
index ce0210c..9cbc955 100644
--- a/src/hid/gtk/gui-drc-window.c
+++ b/src/hid/gtk/gui-drc-window.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -47,8 +45,6 @@
 #define VIOLATION_PIXMAP_PIXEL_BORDER 5
 #define VIOLATION_PIXMAP_PCB_SIZE     MIL_TO_COORD (100)
 
-RCSID ("$Id$");
-
 static GtkWidget *drc_window, *drc_list;
 static GtkListStore *drc_list_model = NULL;
 static int num_violations = 0;
diff --git a/src/hid/gtk/gui-drc-window.h b/src/hid/gtk/gui-drc-window.h
index 8b3558a..367cb8f 100644
--- a/src/hid/gtk/gui-drc-window.h
+++ b/src/hid/gtk/gui-drc-window.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/hid/gtk/gui-icons-misc.data b/src/hid/gtk/gui-icons-misc.data
index 0d121b9..f0d2ec7 100644
--- a/src/hid/gtk/gui-icons-misc.data
+++ b/src/hid/gtk/gui-icons-misc.data
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 
diff --git a/src/hid/gtk/gui-icons-mode-buttons.data b/src/hid/gtk/gui-icons-mode-buttons.data
index be040f1..19e9ce8 100644
--- a/src/hid/gtk/gui-icons-mode-buttons.data
+++ b/src/hid/gtk/gui-icons-mode-buttons.data
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/hid/gtk/gui-keyref-window.c b/src/hid/gtk/gui-keyref-window.c
index 6eaf3eb..e2632ca 100644
--- a/src/hid/gtk/gui-keyref-window.c
+++ b/src/hid/gtk/gui-keyref-window.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -38,8 +36,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static GtkWidget *keyref_window;
 
 
diff --git a/src/hid/gtk/gui-library-window.c b/src/hid/gtk/gui-library-window.c
index 96b5f1f..1d22dd4 100644
--- a/src/hid/gtk/gui-library-window.c
+++ b/src/hid/gtk/gui-library-window.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -69,8 +67,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static GtkWidget *library_window;
 
 #include "gui-pinout-preview.h"
diff --git a/src/hid/gtk/gui-log-window.c b/src/hid/gtk/gui-log-window.c
index 01daaee..46413c1 100644
--- a/src/hid/gtk/gui-log-window.c
+++ b/src/hid/gtk/gui-log-window.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -40,8 +38,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static GtkWidget *log_window, *log_text;
 static gboolean log_show_on_append = FALSE;
 
diff --git a/src/hid/gtk/gui-misc.c b/src/hid/gtk/gui-misc.c
index e84b276..bf3b79e 100644
--- a/src/hid/gtk/gui-misc.c
+++ b/src/hid/gtk/gui-misc.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -43,8 +41,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #define CUSTOM_CURSOR_CLOCKWISE		(GDK_LAST_CURSOR + 10)
 #define CUSTOM_CURSOR_DRAG			(GDK_LAST_CURSOR + 11)
 #define CUSTOM_CURSOR_LOCK			(GDK_LAST_CURSOR + 12)
diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index a847e6d..51192c6 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -64,8 +62,6 @@
 
 #define NET_HIERARCHY_SEPARATOR "/"
 
-RCSID ("$Id$");
-
 static GtkWidget	*netlist_window;
 static GtkWidget	*disable_all_button;
 
diff --git a/src/hid/gtk/gui-pinout-preview.c b/src/hid/gtk/gui-pinout-preview.c
index 79fc0f2..3336945 100644
--- a/src/hid/gtk/gui-pinout-preview.c
+++ b/src/hid/gtk/gui-pinout-preview.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -49,8 +47,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* Just define a sensible scale, lets say (for example), 100 pixel per 150 mil */
 #define SENSIBLE_VIEW_SCALE  (100. / MIL_TO_COORD (150.))
 static void
diff --git a/src/hid/gtk/gui-pinout-preview.h b/src/hid/gtk/gui-pinout-preview.h
index 77fdd1b..1f21e80 100644
--- a/src/hid/gtk/gui-pinout-preview.h
+++ b/src/hid/gtk/gui-pinout-preview.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/hid/gtk/gui-pinout-window.c b/src/hid/gtk/gui-pinout-window.c
index 6433fb1..ea6e8bb 100644
--- a/src/hid/gtk/gui-pinout-window.c
+++ b/src/hid/gtk/gui-pinout-window.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -49,9 +47,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 static void
 pinout_close_cb (GtkWidget * widget, GtkWidget *top_window)
 {
diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index e876220..2335a74 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -128,8 +126,6 @@ a zoom in/out.
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static bool ignore_layer_update;
 
 static GtkWidget *ghid_load_menus (void);
diff --git a/src/hid/gtk/gui-utils.c b/src/hid/gtk/gui-utils.c
index 0fcb5d2..f776f5c 100644
--- a/src/hid/gtk/gui-utils.c
+++ b/src/hid/gtk/gui-utils.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -39,8 +37,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* Not a gui function, but no better place to put it...
  */
 gboolean
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index ea49bee..788553e 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/hid/hidint.h b/src/hid/hidint.h
index b986f78..8d62d6b 100644
--- a/src/hid/hidint.h
+++ b/src/hid/hidint.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /* HID internal interfaces.  These may ONLY be called from the HID
    modules, not from the common PCB code.  */
 
diff --git a/src/hid/lesstif/dialogs.c b/src/hid/lesstif/dialogs.c
index 5d744f9..59a44c8 100644
--- a/src/hid/lesstif/dialogs.c
+++ b/src/hid/lesstif/dialogs.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -26,8 +24,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented GUI function %s\n", __FUNCTION__), abort()
 
 static Arg args[30];
diff --git a/src/hid/lesstif/lesstif.h b/src/hid/lesstif/lesstif.h
index 235bf57..ccbe89b 100644
--- a/src/hid/lesstif/lesstif.h
+++ b/src/hid/lesstif/lesstif.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #define app_context lesstif_app_context
 #define appwidget lesstif_appwidget
 #define display lesstif_display
diff --git a/src/hid/lesstif/library.c b/src/hid/lesstif/library.c
index 029a023..3c8d7d7 100644
--- a/src/hid/lesstif/library.c
+++ b/src/hid/lesstif/library.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -26,8 +24,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static Arg args[30];
 static int n;
 #define stdarg(t,v) XtSetArg(args[n], t, v); n++
diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index c625fc2..f6d3098 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -39,8 +37,6 @@
 
 #include <sys/poll.h>
 
-RCSID ("$Id$");
-
 #ifndef XtRDouble
 #define XtRDouble "Double"
 #endif
diff --git a/src/hid/lesstif/menu.c b/src/hid/lesstif/menu.c
index 8c71c61..d761ce5 100644
--- a/src/hid/lesstif/menu.c
+++ b/src/hid/lesstif/menu.c
@@ -1,5 +1,3 @@
-//* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -31,8 +29,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #ifndef R_OK
 /* Common value for systems that don't define it.  */
 #define R_OK 4
diff --git a/src/hid/lesstif/netlist.c b/src/hid/lesstif/netlist.c
index ce28539..572fba1 100644
--- a/src/hid/lesstif/netlist.c
+++ b/src/hid/lesstif/netlist.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -31,8 +29,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static Arg args[30];
 static int n;
 #define stdarg(t,v) XtSetArg(args[n], t, v); n++
diff --git a/src/hid/lesstif/styles.c b/src/hid/lesstif/styles.c
index 5ec19d8..824b7f3 100644
--- a/src/hid/lesstif/styles.c
+++ b/src/hid/lesstif/styles.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -27,8 +25,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* There are three places where styles are kept:
 
    First, the "active" style is in Settings.LineThickness et al.
diff --git a/src/hid/lesstif/xincludes.h b/src/hid/lesstif/xincludes.h
index af1314c..d8e7b09 100644
--- a/src/hid/lesstif/xincludes.h
+++ b/src/hid/lesstif/xincludes.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  * Some of the X headers are not very friendly in terms of namespace.
  * For example, X.h typedef's Mask but we use Mask in the core of pcb
diff --git a/src/hid/lpr/lpr.c b/src/hid/lpr/lpr.c
index fc9403b..a03b19c 100644
--- a/src/hid/lpr/lpr.c
+++ b/src/hid/lpr/lpr.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -24,8 +22,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented PS function %s.\n", __FUNCTION__); abort()
 
 static HID_Attribute base_lpr_options[] = {
diff --git a/src/hid/nelma/nelma.c b/src/hid/nelma/nelma.c
index 9e75927..fd0aa11 100644
--- a/src/hid/nelma/nelma.c
+++ b/src/hid/nelma/nelma.c
@@ -82,8 +82,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID("$Id$");
-
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented PNG function %s.\n", __FUNCTION__); abort()
 
 /* Needed for PNG export */
diff --git a/src/hid/png/png.h b/src/hid/png/png.h
index e528753..bdd06e1 100644
--- a/src/hid/png/png.h
+++ b/src/hid/png/png.h
@@ -1,3 +1 @@
-/* $Id$ */
-
 extern void png_hid_export_to_file (FILE *, HID_Attr_Val *);
diff --git a/src/hid/ps/eps.c b/src/hid/ps/eps.c
index a406f38..ba27023 100644
--- a/src/hid/ps/eps.c
+++ b/src/hid/ps/eps.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -26,8 +24,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented EPS function %s.\n", __FUNCTION__); abort()
 
 /*----------------------------------------------------------------------------*/
diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 6ecbc64..263a240 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
@@ -30,8 +28,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented PS function %s.\n", __FUNCTION__); abort()
 
 static int ps_set_layer (const char *name, int group, int empty);
diff --git a/src/hid/ps/ps.h b/src/hid/ps/ps.h
index 661eaa8..a6ea013 100644
--- a/src/hid/ps/ps.h
+++ b/src/hid/ps/ps.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 extern HID ps_hid;
 extern void ps_hid_export_to_file (FILE *, HID_Attr_Val *);
 extern void ps_start_file (FILE *);
diff --git a/src/icon.data b/src/icon.data
index 7e55e93..470b2c8 100644
--- a/src/icon.data
+++ b/src/icon.data
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 #define icon_width 32
 #define icon_height 32
diff --git a/src/icons/Makefile.am b/src/icons/Makefile.am
index 23cc5a4..f2c29ff 100644
--- a/src/icons/Makefile.am
+++ b/src/icons/Makefile.am
@@ -1,4 +1,2 @@
-## $Id$
-
 EXTRA_DIST= hand.dat hcurs.dat lcurs.dat lock.dat
 
diff --git a/src/insert.c b/src/insert.c
index 42661ec..7e12874 100644
--- a/src/insert.c
+++ b/src/insert.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -56,11 +54,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
-
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/insert.h b/src/insert.h
index 0476e8d..2793edf 100644
--- a/src/insert.h
+++ b/src/insert.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for inserting points into objects
diff --git a/src/intersect.c b/src/intersect.c
index a3ee03f..33b359d 100644
--- a/src/intersect.c
+++ b/src/intersect.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -50,9 +48,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/intersect.h b/src/intersect.h
index 2c1e7f5..7ed758e 100644
--- a/src/intersect.h
+++ b/src/intersect.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/line.c b/src/line.c
index 22efc2b..6846b2d 100644
--- a/src/line.c
+++ b/src/line.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -48,8 +46,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static double drc_lines (PointTypePtr end, bool way);
 
 /* ---------------------------------------------------------------------------
diff --git a/src/line.h b/src/line.h
index 82db33a..8a5e4d2 100644
--- a/src/line.h
+++ b/src/line.h
@@ -23,7 +23,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for inserting points into objects
diff --git a/src/macro.h b/src/macro.h
index 025351f..59c53ba 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* some commonly used macros not related to a special C-file
diff --git a/src/main.c b/src/main.c
index 48e825c..e170f5a 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -78,9 +76,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 #define PCBLIBPATH ".:" PCBLIBDIR
 
 
diff --git a/src/mirror.c b/src/mirror.c
index c48ebf3..54c53cb 100644
--- a/src/mirror.c
+++ b/src/mirror.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -54,11 +52,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
-
-
 /* ---------------------------------------------------------------------------
  * mirrors the coordinates of an element
  * an additional offset is passed
diff --git a/src/mirror.h b/src/mirror.h
index da61744..50f648a 100644
--- a/src/mirror.h
+++ b/src/mirror.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes to change objects mirror flag
diff --git a/src/misc.c b/src/misc.c
index 3cc04a0..13ce8f6 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -80,9 +78,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 /*	forward declarations	*/
 static char *BumpName (char *);
 static void GetGridLockCoordinates (int, void *, void *, void *,
diff --git a/src/misc.h b/src/misc.h
index a0220c0..2f34cfe 100644
--- a/src/misc.h
+++ b/src/misc.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for misc routines
diff --git a/src/mode_icon.data b/src/mode_icon.data
index e88866e..40e587e 100644
--- a/src/mode_icon.data
+++ b/src/mode_icon.data
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 #define BufferModeIcon_width 21
diff --git a/src/move.c b/src/move.c
index 56ced53..f1971d5 100644
--- a/src/move.c
+++ b/src/move.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -58,11 +56,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
-
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/move.h b/src/move.h
index 3daec3d..801a5cc 100644
--- a/src/move.h
+++ b/src/move.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for move routines
diff --git a/src/mtspace.c b/src/mtspace.c
index a4bb314..8bb3a8c 100644
--- a/src/mtspace.c
+++ b/src/mtspace.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -55,8 +53,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* mtspace data structures are built on r-trees. */
 
 /* ---------------------------------------------------------------------------
diff --git a/src/mtspace.h b/src/mtspace.h
index c169170..baf368f 100644
--- a/src/mtspace.h
+++ b/src/mtspace.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/mymem.c b/src/mymem.c
index 73b3ef7..2dd10a1 100644
--- a/src/mymem.c
+++ b/src/mymem.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -49,8 +47,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------------
  * local prototypes
  */
diff --git a/src/mymem.h b/src/mymem.h
index a66e349..88c5628 100644
--- a/src/mymem.h
+++ b/src/mymem.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for memory routines
diff --git a/src/netlist.c b/src/netlist.c
index 06f0e77..f467f52 100644
--- a/src/netlist.c
+++ b/src/netlist.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -71,8 +69,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /*
   int    PCB->NetlistLib.MenuN
   char * PCB->NetlistLib.Menu[i].Name
diff --git a/src/parse_l.h b/src/parse_l.h
index eb01528..dfeed70 100644
--- a/src/parse_l.h
+++ b/src/parse_l.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* just defines common parser identifiers
diff --git a/src/parse_l.l b/src/parse_l.l
index 33a115d..c0ec6a7 100644
--- a/src/parse_l.l
+++ b/src/parse_l.l
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 %{
 /*
  *                            COPYRIGHT
@@ -48,9 +46,6 @@
 # include <dmalloc.h> /* see http://dmalloc.com */
 #endif
 
-RCSID("$Id$");
-
-
 #include "global.h"
 #include "crosshair.h"
 #include "data.h"
diff --git a/src/parse_y.y b/src/parse_y.y
index 8cf1888..adc2387 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -1,4 +1,3 @@
-/* $Id$ */
 /*
  * ************************** README *******************
  *
@@ -60,8 +59,6 @@
 # include <dmalloc.h> /* see http://dmalloc.com */
 #endif
 
-RCSID("$Id$");
-
 static	LayerTypePtr	Layer;
 static	PolygonTypePtr	Polygon;
 static	SymbolTypePtr	Symbol;
diff --git a/src/pcbtest.sh.in b/src/pcbtest.sh.in
index e1e97b2..516a526 100755
--- a/src/pcbtest.sh.in
+++ b/src/pcbtest.sh.in
@@ -22,9 +22,6 @@
 #   Contact addresses for paper mail and Email:
 #   Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
 #   Thomas.Nau@rz.uni-ulm.de
-# 
-#   RCS: $Id$
-#
 #
 #
 # starts a test installation of pcb
diff --git a/src/polygon.h b/src/polygon.h
index bdbef46..e5d1ace 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for polygon editing routines
diff --git a/src/print.c b/src/print.c
index 4c01721..6d984a6 100644
--- a/src/print.c
+++ b/src/print.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -66,8 +64,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------------
  * prints a FAB drawing.
  */
diff --git a/src/print.h b/src/print.h
index 2215aea..85943f3 100644
--- a/src/print.h
+++ b/src/print.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for printing routines
diff --git a/src/puller.c b/src/puller.c
index 959ac54..90f082a 100644
--- a/src/puller.c
+++ b/src/puller.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -73,8 +71,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 #define abort1() fprintf(stderr, "abort at line %d\n", __LINE__), abort()
 
 #define TRACE0 0
diff --git a/src/rats.c b/src/rats.c
index d2aa8bb..fc6948f 100644
--- a/src/rats.c
+++ b/src/rats.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -61,9 +59,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 #define TRIEDFIRST 0x1
 #define BESTFOUND 0x2
 
diff --git a/src/rats.h b/src/rats.h
index 9479fe0..16e5338 100644
--- a/src/rats.h
+++ b/src/rats.h
@@ -24,7 +24,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for rats routines
diff --git a/src/remove.c b/src/remove.c
index 4554597..d38805f 100644
--- a/src/remove.c
+++ b/src/remove.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -58,9 +56,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/remove.h b/src/remove.h
index 1fc8890..9066428 100644
--- a/src/remove.h
+++ b/src/remove.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for remove routines
diff --git a/src/report.h b/src/report.h
index 6d0184e..c995cde 100644
--- a/src/report.h
+++ b/src/report.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 #ifndef PCB_REPORT_H
diff --git a/src/res_lex.l b/src/res_lex.l
index 5e08666..dd4dadd 100644
--- a/src/res_lex.l
+++ b/src/res_lex.l
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 %{
 
 #ifdef HAVE_CONFIG_H
@@ -18,8 +16,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID("$Id$");
-
 #define YY_NO_INPUT
 #define YY_INPUT(buf,result,max_size) { result = res_parse_getchars(buf, max_size); }
  
diff --git a/src/res_parse.y b/src/res_parse.y
index efc44a0..97c1aae 100644
--- a/src/res_parse.y
+++ b/src/res_parse.y
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 %{
 
 #ifdef HAVE_CONFIG_H
@@ -26,9 +24,6 @@
 #include <dmalloc.h>
 #endif
 
-
-RCSID("$Id$");
-
 static Resource *parsed_res;
 static Resource *current_res;
 
diff --git a/src/resource.h b/src/resource.h
index 119d615..9385e6a 100644
--- a/src/resource.h
+++ b/src/resource.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 #ifndef PCB_RESOURCE_H
 #define PCB_RESOURCE_H
 
diff --git a/src/rotate.c b/src/rotate.c
index 1763406..99f379d 100644
--- a/src/rotate.c
+++ b/src/rotate.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -56,11 +54,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
-
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/rotate.h b/src/rotate.h
index 7748b19..2ac060e 100644
--- a/src/rotate.h
+++ b/src/rotate.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for transform routines
diff --git a/src/rtree.c b/src/rtree.c
index 680be3d..6558427 100644
--- a/src/rtree.c
+++ b/src/rtree.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -53,9 +51,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 #define SLOW_ASSERTS
 /* All rectangles are closed on the bottom left and open on the
  * top right. i.e. they contain one corner point, but not the other.
diff --git a/src/rtree.h b/src/rtree.h
index cb23307..d44de09 100644
--- a/src/rtree.h
+++ b/src/rtree.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/rubberband.c b/src/rubberband.c
index 4fe166c..f5f7442 100644
--- a/src/rubberband.c
+++ b/src/rubberband.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -59,9 +57,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/rubberband.h b/src/rubberband.h
index 9e671bf..e281f3f 100644
--- a/src/rubberband.h
+++ b/src/rubberband.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for rubberband routines
diff --git a/src/search.c b/src/search.c
index 5417462..29271f0 100644
--- a/src/search.c
+++ b/src/search.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -54,9 +52,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
-
 /* ---------------------------------------------------------------------------
  * some local identifiers
  */
diff --git a/src/search.h b/src/search.h
index b1a6196..30553dd 100644
--- a/src/search.h
+++ b/src/search.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for search routines
diff --git a/src/select.c b/src/select.c
index 96db234..31ea28d 100644
--- a/src/select.c
+++ b/src/select.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -59,8 +57,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------------
  * toggles the selection of any kind of object
  * the different types are defined by search.h
diff --git a/src/select.h b/src/select.h
index f3edde6..6d91278 100644
--- a/src/select.h
+++ b/src/select.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for select routines
diff --git a/src/set.c b/src/set.c
index 18fffa7..21351da 100644
--- a/src/set.c
+++ b/src/set.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -62,8 +60,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static int mode_position = 0;
 static int mode_stack[MAX_MODESTACK_DEPTH];
 
diff --git a/src/set.h b/src/set.h
index 4bfe6ec..d3b9a1f 100644
--- a/src/set.h
+++ b/src/set.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for update routines
diff --git a/src/strflags.c b/src/strflags.c
index 8f4912c..3699c53 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -1,4 +1,3 @@
-/* $Id$ */
 /*
  *                            COPYRIGHT
  *
@@ -54,8 +53,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* Because all the macros expect it, that's why.  */
 typedef struct
 {
diff --git a/src/strflags.h b/src/strflags.h
index da16b64..a5f5b44 100644
--- a/src/strflags.h
+++ b/src/strflags.h
@@ -1,4 +1,3 @@
-/* $Id$ */
 /*
  *                            COPYRIGHT
  *
diff --git a/src/thermal.c b/src/thermal.c
index 03fee24..d2cfa95 100644
--- a/src/thermal.c
+++ b/src/thermal.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -75,8 +73,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static PCBTypePtr pcb;
 
 struct cent
diff --git a/src/thermal.h b/src/thermal.h
index 67c04e0..b252f4f 100644
--- a/src/thermal.h
+++ b/src/thermal.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for thermal routines
diff --git a/src/undo.c b/src/undo.c
index 17bebb9..87e02aa 100644
--- a/src/undo.c
+++ b/src/undo.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -74,8 +72,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static bool between_increment_and_restore = false;
 static bool added_undo_between_increment_and_restore = false;
 
diff --git a/src/undo.h b/src/undo.h
index a6cd324..2ec6442 100644
--- a/src/undo.h
+++ b/src/undo.h
@@ -22,7 +22,6 @@
  *  Thomas Nau, Schlehenweg 15, 88471 Baustetten, Germany
  *  Thomas.Nau@rz.uni-ulm.de
  *
- *  RCS: $Id$
  */
 
 /* prototypes for undo routines
diff --git a/src/vector.c b/src/vector.c
index c039b89..aad1c62 100644
--- a/src/vector.c
+++ b/src/vector.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -51,8 +49,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 /* ---------------------------------------------------------------------------
  * some local prototypes
  */
diff --git a/src/vector.h b/src/vector.h
index 344118d..b6b6de1 100644
--- a/src/vector.h
+++ b/src/vector.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/vendor.c b/src/vendor.c
index 2abcca0..f7b89f7 100644
--- a/src/vendor.c
+++ b/src/vendor.c
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
@@ -64,8 +62,6 @@
 #include <dmalloc.h>
 #endif
 
-RCSID ("$Id$");
-
 static void add_to_drills (char *);
 static void apply_vendor_map (void);
 static void process_skips (Resource *);
diff --git a/src/vendor.h b/src/vendor.h
index aba4148..9706448 100644
--- a/src/vendor.h
+++ b/src/vendor.h
@@ -1,5 +1,3 @@
-/* $Id$ */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/tests/run_tests.sh b/tests/run_tests.sh
index 5bd4acb..b24bc65 100755
--- a/tests/run_tests.sh
+++ b/tests/run_tests.sh
@@ -1,7 +1,5 @@
 #!/bin/sh
 #
-# $Id: run_tests.sh,v 1.8 2009/02/17 00:42:31 danmc Exp $
-#
 # Copyright (c) 2003, 2004, 2005, 2006, 2007, 2009, 2010 Dan McMahill
 
 #  This program is free software; you can redistribute it and/or modify
@@ -334,7 +332,6 @@ compare_bom() {
 
     # an example BOM file is:
 
-    #  # $Id$
     #  # PcbBOM Version 1.0
     #  # Date: Wed Jun 17 14:41:43 2009 UTC
     #  # Author: Dan McMahill
diff --git a/tools/Makefile.am b/tools/Makefile.am
index f350471..3900614 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -1,6 +1,3 @@
-## $Id$
-##
-
 ## XXX the tools should probably be cleaned up a bit and installed
 ## in ${exec_prefix}/bin  (by bin_SCRIPTS and bin_PROGRAMS)
 
diff --git a/tools/MergePCBPS b/tools/MergePCBPS
index 8785d28..8d63529 100755
--- a/tools/MergePCBPS
+++ b/tools/MergePCBPS
@@ -1,7 +1,5 @@
 #!/bin/sh
 #
-# $Id$
-#
 # merges PostScript files created by pcb (version >= 1.4.2) into
 # a single file for backward compatibility
 #
diff --git a/tools/Merge_dimPCBPS b/tools/Merge_dimPCBPS
index 08c559e..84ce3ab 100755
--- a/tools/Merge_dimPCBPS
+++ b/tools/Merge_dimPCBPS
@@ -1,7 +1,5 @@
 #!/bin/sh
 #
-# $Id$
-#
 # merges PostScript files created by pcb (version >= 1.4.2) into
 # a single file for backward compatibility
 #
diff --git a/tutorial/Makefile.am b/tutorial/Makefile.am
index 85daeac..2b9320f 100644
--- a/tutorial/Makefile.am
+++ b/tutorial/Makefile.am
@@ -1,5 +1,3 @@
-## $Id$
-
 tutdir=	@docdir@/tutorial
 tut_DATA=	${TUTORIALS}
